spec/clearance/session_spec.rb

```diff
@@ -30,6 +30,17 @@
     expect(session.current_user).to be_nil
   end
 
+  it 'sets a custom cookie name' do
+    Clearance.configuration.cookie_domain = 'custom_token'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.